lib/api/controllers/routerController.js

```diff
@@ -145,7 +144,7 @@ function RouterController (kuzzle) {
      * @param request
      * @param response
      */
-    handleRoute = function(request, response) {
+    function handleRoute (request, response) {
```

Comment:
Function declarations inside blocks are invalid according to ECMA specifications.
Consider assigning this one to a variable or, better yet, promote this snippet of code as a real function of its own.